### PR TITLE
[circlechef] Update proto for BCQ operations

### PR DESCRIPTION
**Pull request by llFreetimell**
Parent Issue : #1517
Draft : #2739

This commit will update `circlechef.proto` to include BCQ operations.

ONE-DCO-1.0-Signed-off-by: Seok NamKoong <seok9311@naver.com>

### CODE REVIEW

File: compiler/circlechef/proto/circlechef.proto [hunk 1]
```diff
@@ -68,13 +68,25 @@ message InstanceNormOptions {
   optional Activation activation = 2 [default = NONE];
 }
 
+message BCQFullyConnectedOptions {
+  optional int32 weights_hidden_size = 1 [default = 0];
+  optional Activation activation = 2 [default = NONE];
```
Comment (seanshpark) [inline compiler/circlechef/proto/circlechef.proto:73]: usually I've met activation in the first. is it OK to swap these two parameters?

File: compiler/circlechef/proto/circlechef.proto [hunk 2]
```diff
@@ -68,13 +68,25 @@ message InstanceNormOptions {
...
+}
+
+message BCQGatherOptions {
+  optional int32 input_hidden_size = 1 [default = 0];
+  optional int32 axis = 2 [default = 0];
```
Comment (seanshpark) [inline compiler/circlechef/proto/circlechef.proto:78]: ditto, is axis first then input_hidden_size OK?

File: compiler/circlechef/proto/circlechef.proto (hunk 1 above)
Comment (llFreetimell) [reply compiler/circlechef/proto/circlechef.proto:73]: I followed below order in circle schema not to be confused.

https://github.com/Samsung/ONE/blob/bd60a8ed0f25786d91e1949c810e624887a49bb7/nnpackage/schema/circle_schema.fbs#L991-L999

And `InstanceNormOptions` in this file also have activation as second.
Nevertheless if you don't like this, I will change it :)

Comment (seanshpark) [reply compiler/circlechef/proto/circlechef.proto:73]: Ah.. let's follow source circle schema :)

**Pull request merged**